packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -64,59 +108,157 @@ Tests can be executed in any of the following supported form factors:
-- {string} title - The `describe` block title.
-- {string[]} viewports - The list of Terra viewports to test.
-- {function} - the test function to execute for each viewport.
+To extend the default configuration create a `wdio.conf.js` file at the root of your project and apply the desired options.
 
+wdio.conf.js
 ```js
-Terra.describeViewports('Describe viewports', ['tiny', 'huge'], () => {
-  it('should test tiny and huge screens', () => {
-    browser.url('/testing/route/');
-    Terra.validates.screenshot('describe viewports');
-  });
-});
+const { config } = require('@cerner/terra-functional-testing');
+
+// Stop the test run if there is a test failure.
+config.bail = 1;
+
+exports.config = config;
 ```
 
-### Terra.describeTests
+## Service Options
+
+Service options are configurable by extending the wdio.conf.js file. These options are applied for every test run.
+
+### selector
+
+Specifies the default element to be captured when taking a screenshot.
+
+Type: `string`
 
-The `Terra.describeTests` helper extends the `Terra.describeViewports` helper by additionally filter tests by locales and themes. All tests within this helper will be executed only if each of the form factors, locales, and themes listed in the options match those defined in the configurations for the current test run. If no form factors, locales, or themes are provided as options, then all tests under this helper will qualify to run in any form factor, locale, or theme. It is recommended to use `Terra.describeTests` over `Terra.describeViewports` because the same features in `Terra.describeViewports` can be achieved with `Terra.describeTests` by eliminating locales and themes and only specify the form factors. This helper accepts these arguments in the following order:
-- {string} title - The describe block title.
-- {Object} options - An object containing arrays of formFactors, locales, and themes that the block of tests will only qualify to execute in.
-- {function} - The block of tests to execute against each viewport.
+Required: `false`
 
+Default: `[data-terra-test-content] *:first-child`
 
-The below describe block qualifies to run because each of the form factors, locales, and themes provided to `describeTests` match the CLI options specified in the test runner.
-```sh
-terra wdio --formFactors tiny --locales en --themes terra-default-theme
+<Notice variant="important" ariaLevel="3">
+  This selector is used as the default screenshot selector for Terra.validates.element and Terra.validates.screenshot.
+</Notice>
+
+```js
+const { config } = require('@cerner/terra-functional-testing');
+
+config.serviceOptions = {
+  selector: '#root',
+};
+
+exports.config = config;
 ```
 
+### seleniumVersion
+
+Specifies the [docker selenium](https://github.com/SeleniumHQ/docker-selenium/tree/selenium-3) version used within the docker container.
```

Comment:
Should we just link to https://github.com/SeleniumHQ/docker-selenium instead of their active branch? Not sure if they would delete their `selenium-3` branch in the future.

Reply:
I'm not too strongly opinionated here. We're currently on grid 3. The homepage is on Grid 4. I pulled this link for their homepage:

https://github.com/SeleniumHQ/docker-selenium#point_right-status-grid-4-is-under-development-and-on-a-beta-stage

![image](https://user-images.githubusercontent.com/6720991/112537914-3edc8580-8d7d-11eb-9ee9-bb563395a7bf.png)

Reply:
Ah, ok. I don't care either way. Let's leave it as is.